app/src/main/java/org/simple/clinic/teleconsultlog/success/TeleConsultSuccessScreenInjector.kt

```diff
@@ -0,0 +1,5 @@
+package org.simple.clinic.teleconsultlog.success
+
+interface TeleConsultSuccessScreenInjector {
```

Comment:
We have lately been following the convention of making this interface a nested interface of the screen that needs injection and skipping the screen name prefix. Can we follow that here please?